### DISCUSSION TITLE
Hello Boris

### DISCUSSION THREAD

**Discussion by vladyslaveski** [boris-kz/CogAlg#81] [General]
Nice to meet you.
I want you to discuss the project in detail.

**Comment by boris-kz** [comment]
Hi Vlady, what sort of details do you want?

**Reply by vladyslaveski** [reply]
I got your project in freelancer.
But I don't know what you want to me.
so, I hope you let me know the project details , what you want, what should I do for you.

**Comment by boris-kz** [comment]
Thanks for taking a look, but if my readme and wiki don't even begin to explain this to you, then this project is not for you.
